### DISCUSSION TITLE
Hot key list should be constant.

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#709] [Ideas]
![image](https://user-images.githubusercontent.com/6903107/135072451-72483332-7601-4ec6-bd53-cffe9844d260.png)

- Shortcut keys are no longer displayed if the maximum number exceeds. If you go down afterwards, the shortcut is not displayed. The lower part cannot be used as a shortcut key, and the design is strange.
- I think it's better to keep the shortcut key fixed. It should run match position list item.
- so always showing hotkey list. if some item scrolling change to ctrl 2 to ctrl 1, hotkey launch match position item.

**Comment by deefrawley** [comment]
I like that idea. What would happen if you changed your settings to show more than 10 results though?

**Reply by onesounds** [reply]
I didn't think about it! 😂 because I didn't use it like that. It is also questionable whether more than 10 are needed.

**Reply by deefrawley** [reply]
Flow settings allow for up to 17

**Reply by onesounds** [reply]
- Can't we limit it to 9? If there are more than 10, it is difficult to find out where and what shortcut is right.
- or If there are more than 10 or more, another shortcut is assigned. (e.g 1~9 = ctrl, 10~19 alt...) 

Come to think of it, we just need to allocate the top nine. The concept of executing the same fixed position is fine. If you see 20, you can see the shortcut keys on the top nine on the screen.

**Reply by deefrawley** [reply]
You could move to the QWERTY keys at more than 10 results - Ctrl+Q, Ctrl+W etc. To cater to everyone you could have an option to use other keyboard formats as well with the aim being to use the next row under the number keys.

**Reply by onesounds** [reply]
> You could move to the QWERTY keys at more than 10 results - Ctrl+Q, Ctrl+W etc. To cater to everyone you could have an option to use other keyboard formats as well with the aim being to use the next row under the number keys.

I think that would be right. We have a function key. ctrl+f1~f12 waiting for pressed by us.
